perception/mil_mlp/README.md

```diff
@@ -0,0 +1,168 @@
+# Transfer Learning Object Detection - A Short Tutorial
```

Comment:
Before jumping into usage, nice to provide a description of what this package is / does, you can even just copy this from package.xml